### PR TITLE
Regional satellite page updates

### PR DESCRIPTION
**Pull request by leewujung**
This PR is WIP and not yet ready for review.

### CODE REVIEW

**Comment by leewujung** [comment]
@abkfenris : seems that build/deploy failed due to some environment problem, but I was able to build locally. Could you please look into that? Thanks!!

**Comment by abkfenris** [comment]
Those failures are fine right now. We are intentionally leaving them broken for the moment, until we are ready to launch the new site. 

#113 takes care of that when we are ready to flip the switch.

With the change to Northwest/Southwest, how about changing Maine to Northeast?

File: ohw22/satellites.md [hunk 1]
```diff
@@ -64,4 +66,4 @@ For help with timezone conversions, see [https://www.timeanddate.com](https://ww
 :hidden:
 
 maine/index
```
Comment (abkfenris) [inline ohw22/satellites.md:68]: This should add the satellite to the menu in the sidebar, and get the menu to show up on it's page too.

```suggestion
maine/index
seattle/index
```

**Comment by emiliom** [comment]
The preview built fine on netlify.

File: ohw22/satellites.md (hunk 1 above)
Comment (emiliom) [reply ohw22/satellites.md:68]: That's what we want. I'll commit your suggestion

Comment (leewujung) [reply ohw22/satellites.md:68]: Awesome, thanks!

**Comment by leewujung** [comment]
> Those failures are fine right now. We are intentionally leaving them broken for the moment, until we are ready to launch the new site.
> 
> #113 takes care of that when we are ready to flip the switch.

Ah I see. That's cool!

> With the change to Northwest/Southwest, how about changing Maine to Northeast?

Sounds good. Will do.

File: ohw22/seattle/index.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+**Contact**
+
+For questions specific to the regular OHW satellite, please email Dr. Wu-Jung Lee (leewj@uw.edu).
+
+
+## Undergraduate summer program (August 8-26, 2022)
+
+This "Data Science in Oceanography" undergraduate program will provide opportunities for undergraduate students in data-driven research in oceanography. Participating students will interact closely with faculty and graduate student mentors to develop and advance individual research projects. Students will fully participate in the OHW Northwest satellite event as part of this longer undergraduate program.
+
+See [details in the program flyer](./undergrad_flyer.pdf)!
```
Comment (leewujung) [inline ohw22/seattle/index.md:41]: Somehow this doesn't work, but I also don't know much about jekyll. Help!

Comment (leewujung) [inline ohw22/seattle/index.md:41]: Somehow this link doesn't work, but I also don't know much about jekyll. Help!

Comment (emiliom) [reply ohw22/seattle/index.md:41]: I'll get to it right now. BTW, we're using Sphinx now, not Jekyll.

Comment (leewujung) [reply ohw22/seattle/index.md:41]: Oh funny, somehow i saw that jekyll wording somewhere. If you can push a fix here directly that'll be great. Thanks!

Comment (emiliom) [reply ohw22/seattle/index.md:41]: There are some jekyll words and warnings from the afterlife still floating around. Ignore them :wink:

Comment (emiliom) [reply ohw22/seattle/index.md:41]: Hmm. Sphinx refuses to build a link when the reference is not a full url (http ...). I suspect Sphinx doesn't like non-docs files mixed in with the docs files. I tested it with a regular png, with the same result.

My quick solution will be to move the pdf to the assets folder. Working on it

Comment (abkfenris) [inline ohw22/seattle/index.md:41]:
```suggestion
See {download}`details in the program flyer <./undergrad_flyer.pdf>`
```

It needs to be a Sphinx role to say that it's downloadable to get Sphinx to move it and not try to parse it.

Comment (emiliom) [reply ohw22/seattle/index.md:41]: Yeah, I was running out of options. I tried [this old solution](https://stackoverflow.com/questions/38613647/sphinx-linking-to-embedded-binary-files-pdfs) to treat it like a normal link and open it in the browser but it still wasn't working. Yikes. I give up, let's go with your suggestion.

Comment (ocefpaf) [reply ohw22/seattle/index.md:41]: Can sphinx read the .md file without myst? Maybe we should add MyST to extensions so we can write markdown.

**Comment by emiliom** [comment]
@leewujung how about we merge this PR now? It'll be helpful to have some of your edits merged. You/we can keep working on edits to the NW satellite event page via a separate PR, if needed.

**Comment by leewujung** [comment]
Yeah, I was gonna say the same thing. Need to attend to other things for next few hours, and then get back to try to make the regional satellite page more complete based on what I know. Will work on the application-related stuff after @paigem's changes.

**Pull request merged**